core/amber/src/main/scala/edu/uci/ics/amber/engine/common/ambermessage/neo/InternalMessage.scala

```diff
@@ -0,0 +1,8 @@
+package edu.uci.ics.amber.engine.common.ambermessage.neo
+
+import edu.uci.ics.amber.engine.common.ambertag.neo.Identifier
+
+trait InternalMessage extends Serializable {
```

Comment:
InternalMessage is a very generic term. Not easy to understand `internal to what` - actor, system..?? Any reason why you changed it from WorkflowMessage?